src/BuildIntegration/Microsoft.NETCore.Native.Windows.props

```diff
@@ -77,9 +77,16 @@ See the LICENSE file in the project root for more information.
       <LinkerArg Include="/OPT:ICF" />
     </ItemGroup>
 
-    <Exec Command="where /Q $(CppLinker)" IgnoreExitCode="true">
-      <Output TaskParameter="ExitCode" PropertyName="_WhereLinker"/>
+    <Exec Condition=" !Exists($(CppCompiler)) OR !Exists($(CppLinker)) " Command="getvctoolspath.bat x86_amd64" LogStandardErrorAsError="true" ContinueOnError="false" ConsoleToMSBuild="true">
```

Comment:
Does `Exist(...)` look for the file on PATH? I think this should be always false here since CppCompiler is just `cl`.